pkg/oc/clusterup/coreinstall/kubeapiserver/master_config.go

```diff
@@ -0,0 +1,64 @@
+package kubeapiserver
```

Comment:
this part and it's callers I like.  Minus the weird deps created (cluster add back into cluster up??)

Reply:
that dep is safe to me (if we move cluster up outside, cluster add will come with it)...